build-a-brain/textbook.rst

```diff
@@ -0,0 +1,157 @@
+|            | 6.4--6.6       |   plasticity (STDP)          | that learns an     |
+|            |                | * Reinforcement learning     | arbitrary 1D       |
+|            |                | * Learning transformations   | transformation     |
+|            |                |   with the hPES rule         |                    |
++------------+----------------+------------------------------+--------------------+
+| Week 15    | Sections       | * A review and overview      | Build a question   |
+|            | 7.1--7.3       |   of the SPA                 | answering model    |
+|            |                | * The Spaun model and tasks  | with control       |
++------------+----------------+------------------------------+--------------------+
+| Week 16    | Section 7.4    | * Probabilistic models       | Build a model      |
+| (optional) |                | * Interpreting the SPA       | to do a simple     |
+|            |                |   as a probabilistic model   | statistical        |
+|            |                |                              | inference problem  |
++------------+----------------+------------------------------+--------------------+
+| Week 17    | Chapter 8      | * Evaluating cognitive       | Begin a course     |
+|            |                |   theories                   | project, using     |
+|            |                | * Detailed discussion of the | tutorial 8         |
+|            |                |   core cognitive criteria    | as an example      |
++------------+----------------+------------------------------+--------------------+
+| Week 18    | Chapter 9      | * A survey of other          | Course project     |
+|            |                |   approaches to cognitive    |                    |
+|            |                |   modelling                  |                    |
+|            |                | * A comparison of the SPA    |                    |
+|            |                |   with past approaches       |                    |
++------------+----------------+------------------------------+--------------------+
+| Week 19    | Chapter 10     | * Conceptual consequences of | Course project     |
+|            |                |   the SPA and NEF methods    |                    |
+|            |                | * Future challenges for this |                    |
+|            |                |   and other approaches       |                    |
++------------+----------------+------------------------------+--------------------+
+
+Assignment details
+==================
+
+The assignments above mirror those in the book.
+It is also useful to ask questions
+about the functioning of the models being built,
+and to ask students to change the models slightly
+to examine different aspects of neural/cognitive processing.
+The intent of this section of the website
+is to provide examples of such questions,
+as we develop them.
+Suggestions welcome.
+
+Suggested projects
```

Comment:
Should we link to our "modelling ideas" repository here? Some of these suggestions are out of date and a bit weird.